Lab1ForReview/Lab1b/Syntax.in.txt

```diff
@@ -0,0 +1,25 @@
+<program> ::= <decllist> "|" <cmpdstmt> "|"
+<decllist> ::= <declaration | <declaration> "|" <decllist>
+<declaration> ::= IDENTIFIER "space" <type> 
+<type1> ::= "nr" | "NR" |"BOOLEAN"| "string" 
+<arraydecl> ::= <type1> "{" nr "}" //numai recursiv
+<type> ::= <type1> | <arraydecl> 
+<cmpdstmt> ::= <stmtlist> 
+
+<stmtlist> ::= <stmt> | <stmt> "|" stmtlist 
+<stmt> ::= <simplstmt> | <structstmt> 
+<simplstmt> ::= <assignstmt> | <iostmt> | <retstmt>
+<assignstmt> ::= <expression> "=" IDENTIFIER
```

Comment:
Identifier was defined in lexic as identifier and in syntax it is written IDENTIFIER, also it should be written between <> if you are using BNF.